src/docs/arch-storage-extension.dox

```diff
@@ -82,24 +84,25 @@ over the network, thus mitigating network latency.
 - When a S3 file handle is created, the local cache gets a copy of the S3 object if
 it doesn't already have it. Since the local cache is a directory on the WiredTiger database's file
 system, the file is accessed using a \c WT_FILE_HANDLE implementation native to WiredTiger.
-Hence, a copy of native \c WT_FILE_HANDLE is kept encapsulated in the S3 file handle.
+Hence, a \c WT_FILE_HANDLE is kept encapsulated in the S3 file handle.
 
 - The S3 store has created a logging class which extends the AWS SDK's logging system.
 
 @section azure_gcp_store Azure and GCP Storage Source
-The Azure and Google Cloud Platform (GCP) store implement the following functionalities:
+The Azure and Google Cloud Platform (GCP) stores implement the following functionality:
 
-- GCP authentication requires setting the \c GOOGLE_APPLICATION_CREDENTIALS environment variable
-which contains the path to the authentication file. Azure authentication requires the
-\c AZURE_STORAGE_CONNECTION_STRING environment variable to be set.
+- The Azure and GCP store follows different authentication methods to connect to their respective
+SDKs. GCP requires setting a \c GOOGLE_APPLICATION_CREDENTIALS environment variable expecting the
+path to an authentication file and Azure requires the \c AZURE_STORAGE_CONNECTION_STRING environment
+variable to be set.
 
 - Another reference count is incremented each time the file handle reference is opened and
 decremented with the close. When the reference count goes zero, the file system is de-registered
 from the storage source and any allocated resources are freed.
 
-- Whenever a GCP or Azure file handle is created, the expectation is that the file does not
-exist in the local file system. All required file handle operations will directly interface with
-the connection class.
+- Whenever a GCP or Azure file handle is created, it does not require the file to be present in the
+local file system. The file directly interactions with the connection class to handle all file
```

Comment:
I think this new sentence needs some clarification, I don't really understand what it's trying to say.